src/cloudai/workloads/ai_dynamo/ai_dynamo.sh

```
@@ -495,6 +537,15 @@ _gpu_list_for_worker() {
   echo "$(echo $CUDA_VISIBLE_DEVICES | cut -d',' -f${start}-${end})"
 }
 
+_gpu_list_for_worker_offset() {
+  local per_worker=$1
+  local idx=$2
+  local offset=${3:-0}
+  local start=$(( 1 + offset + (idx * per_worker) ))
+  local end=$(( start + per_worker - 1 ))
+  echo "$(echo $CUDA_VISIBLE_DEVICES | cut -d',' -f${start}-${end})"
```

Shellcheck (0.11.0)

[style] Useless echo? Instead of 'echo $(cmd)', just use 'cmd'.

(SC2005)